tests/network/libs/nodes.py

```
@@ -0,0 +1,23 @@
+from libs.vm.vm import BaseVirtualMachine
+
+HOSTNAME_LABEL: Final[str] = "kubernetes.io/hostname"
+RHCOS_9_VERSION_PREFIX: Final[str] = "Red Hat Enterprise Linux CoreOS 9"
+RHCOS_10_VERSION_PREFIX: Final[str] = "Red Hat Enterprise Linux CoreOS 10"
+
+
+def node_by_rhcos_version(workers: list[Node], rhcos_version_prefix: str) -> Node:
+    for node in workers:
+        if node.instance.status.nodeInfo.osImage.startswith(rhcos_version_prefix):
+            return node
+    raise ValueError(f"No worker node found with RHCOS version prefix: {rhcos_version_prefix!r}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)